simple_history/management/commands/_populate_utils.py

```diff
@@ -17,13 +17,15 @@ def get_history_model_for_model(model):
 
 def bulk_history_create(model, history_model, batch_size):
     """Save a copy of all instances to the historical model."""
+    excluded_fields = set(history_model.excluded_fields)
     historical_instances = [
         history_model(
             history_date=getattr(instance, '_history_date', now()),
             history_user=getattr(instance, '_history_user', None),
             **{
                 field.attname: getattr(instance, field.attname)
                 for field in instance._meta.fields
+                if not excluded_fields & {field.name, field.attname}
```

Comment:
Why use `field.attname` here? In `fields_included` we only check on `field.name`: https://github.com/treyhunner/django-simple-history/blob/master/simple_history/models.py#L142

Reply:
@rossmechanic Ok, I've updated the code to only have check on `field.name`